assets/quick-add.css

```diff
@@ -122,6 +134,11 @@
   padding-left: 1rem;
 }
 
+.quick-add-modal__content-info--bulk-details > a:hover {
+  color: rgb(var(--color-foreground));
+  text-decoration: underline;
+}
+
 @media screen and (min-width: 990px) {
   .quick-add-modal__content-info--bulk .quick-add__product-media,
   .quick-add-modal__content-info--bulk .quick-add__product-container,
```

Comment:
>Quick add bulk modal header spacing is off

This could be fixed in here. It's a tricky visual bug to replicate but it's due to a media query not using the right value in the file. 

We have: 
```css
@media screen and (min-width: 989px) {
```
and 
```css
@media screen and (max-width: 990px) {
```

We should be using those values the other way around like in the rest of the CSS files. `990` is always used with `min-width` and `989` with `max-width`. 

[Recording](https://screenshot.click/28-17-cqjzi-3qqkb.mp4).
We actually have another instance of this in the file as well that mess with the [padding applied](https://screenshot.click/28-23-suppx-vp3wk.png). 

You can search for `990px` and make sure none is tied to `max-width:` and same for `989` not being tied with `min-width:`